packages/common/src/models/Collection.ts

```diff
@@ -50,7 +50,7 @@ export type CollectionMetadata = {
   cover_art: CID | null
   cover_art_sizes: Nullable<CID>
   cover_art_cids?: Nullable<CoverArtSizesCids>
-  permalink?: string
+  permalink: string
```

Comment:
Whats the context on making this required?

Reply:
the link component complained it might be undefined, and at this point i dont think that is possible, so went ahead and updated everywhere